server/db/populate-db.sql

```diff
@@ -1,4 +1,4 @@
 -- Users seed data
 -- Note: Do NOT add passwords you use in real life. Passwords are now saved as clear text. This is very bad from security point of view. We will have a task to encrypt the passwords
-INSERT INTO users (email, password) values ('admin@cyf.org', 'admin_password');
-INSERT INTO users (email, password) values ('user@cyf.org', 'user_password');
+INSERT INTO users (email,role, password) values ('admin@cyf.org', 'applicant','admin_password');
+INSERT INTO users (email, role, password) values ('user@cyf.org', 'applicant','user_password');
```

Comment:
```suggestion
INSERT INTO users (email, role, password) values ('admin@cyf.org', 'applicant', 'admin_password');
INSERT INTO users (email, role, password) values ('user@cyf.org', 'applicant', 'user_password');
```

Some software teams care about consistent code style, and one part of this is ensuring that there is always a space after a comma. For this project, I don't think this needs to be mandatory but it's always good to keep it in mind when writing and reviewing your code!